frontend/app/helpers/spree/frontend_helper.rb

```diff
@@ -310,9 +310,9 @@ def available_properties_cache_key
 
     def available_properties
       @available_properties ||= Rails.cache.fetch("available-properties/#{available_properties_cache_key}") do
```

Comment:
@mad-eel we will need to also adjust `available_properties_cache_key` - currently it's always the same for the entire app - I think this should take into consideration:

```ruby
[
   Spree::Property.filterable.maximum(:updated_at),
   Spree::ProductProperty.filterable.maximum(:updated_at),
   products_for_filters.maximum(:updated_at)
]
```

and similarly this needs to be adjusted for `available_option_types_cache_key`